src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -2,0 +1,22 @@
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/getUserById';
```

Comment:
[CODE KNOWLEDGE] - If you are using a non-mutating array method, you **don't** need to create a copy of the array. Directly apply the 'filter' method on the 'cats' array.